docs/templates/jenkins/scan-repository.jenkinsfile

```diff
@@ -50,6 +50,11 @@ pipeline {
         // The 'frogbot' executable and other tools it needs will be downloaded through this repository.
         // JF_RELEASES_REPO= ""
 
+        // [Optional]
+        // Avoid printing additional output in pull request comments
+        // For example, a JFrog Advanced Security message in pull request comments.
```

Comment:
// Avoid printing additional output in pull request comments
// For example, a JFrog Advanced Security message in pull request comments.
-->
// Avoid adding extra info to pull request comments. that isn't related to the scan findings.